src/components/PrimaryButton.jsx

```diff
@@ -0,0 +1,28 @@
+import styled from "@emotion/styled";
+
+const StyledButton = styled.button`
+  background-color: ${(props) =>  props.theme.colors.violet};
+  border: none;
+  color: ${(props) => props.theme.colors.white};
+  padding: ${(props) => props.theme.layout.space200}
+    ${(props) => props.theme.layout.space500};
+  text-align: center;
+  text-decoration: none;
+  font-size: 16px;
+  cursor: pointer;
+  border-radius: 5px;
+
+  &:hover {
+    background-color: ${(props) =>  props.theme.colors.lightViolet};
```

Comment:
Can you create the color variables in `light.js` similar to the SecondaryButton component?